phpinsights.php

```diff
@@ -85,4 +85,21 @@
             ],
         ],
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Requirements
+    |--------------------------------------------------------------------------
+    |
+    | Here you may define a level you want to reach per `Insights` category. If
+    | the minimum level defined is not reached, the console will return an exit
+    | error code. All of them are optional and may be individually configured.
```

Comment:
I remember @nunomaduro said me once that each line should be 3 chars shorter than the last one like in Laravel.

Cf [here](https://github.com/nunomaduro/phpinsights/pull/265#discussion_r324462951)